src/native/_auth.js

```diff
@@ -143,7 +143,6 @@ const makeAppInfo = (appInfo) => {
 
 const translateXorName = (str) => {
   const b = new Buffer(str);
-  if (b.length != 32) throw Error("XOR Names _must be_ 32 bytes long.")
```

Comment:
Same comment about this in `translatePrivMDInput` function would be applicable there as the user wouldn't know which param is wrong from the permissions array...??

Reply:
This error won't be necessary anymore as it will have been caught by [validateMDataPermissions](https://github.com/maidsafe/safe_app_nodejs/pull/208/files#diff-c9616fc06535a5565aa65b19f33fa08aR182).

Reply:
I don't see that `validateMDataPermissions` is in any possible execution flow of a call to this `translateXorName` function